packages/big-design/src/components/Button/spec.tsx

```diff
@@ -187,3 +187,25 @@ test('render only icon only with left and right icons button', () => {
 
   expect(screen.getAllByTestId('icon-only')).toHaveLength(1);
 });
+
+test('hides content when isLoading is true', () => {
+  const plusIcon = <AddIcon data-testid="icon-only" role="icon" aria-hidden="false" />;
```

Comment:
Had to add the `role=icon` so I could grab it from the DOM, and be able to test if the content was visible or not since using `getByText('Button')` the content appeared in the DOM 🤔 even though `isLoading` was true.

Reply:
If you're using `role` do we need `data-testid`? Otherwise can you use `getByTestId('icon-only')` instead of `role`?

Hmm, maybe the `Button` label is only being visually hidden but remains available for DOM/accessibility?

Reply:
ohh yes! Good catch, it shoulnd't have the data-testid 😅 . I copied and pasted `<AddIcon data-testid="icon-only" role="icon" aria-hidden="false" />;` from a previous test, but I forgot to delete the things that I didn't need for this test. 

Yes, I think the selectors that works for accessibility are the ones that uses `*ByRole`